broadinstitute/pearl
# [JN-477] admin help for export

This adds basic help docs (copied over from Pepper) for the download capability.  The existence of help docs within our app is probably temporary.  Long-term, we'll want to use ZenDesk or some other solution that lets non-engineers write/edit the docs.   I'll put in a request to comms to see about getting that set up.  In the meantime, this will give us a place to park user-visible documentation.  Also, having it be in-app makes the release/review process much simpler -- the docs get auto-released with the features, rather than having to be reviewed and released separately.

<img width="584" alt="image" src="https://github.com/broadinstitute/juniper/assets/2800795/7fb3a649-0941-4b6c-82fd-496504c1d084">

<img width="1161" alt="image" src="https://github.com/broadinstitute/juniper/assets/2800795/f3f4816a-d7d8-4471-bcf5-2eba5416bbe5">


TO TEST:
1. Go to https://localhost:3000/ourhealth/env/live/participants
2. click "download"
3. click the link to the help page
4. confirm the help page shows up in a new tab
